src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -577,6 +579,13 @@ public T Attach<T>(JsonRpcProxyOptions options)
             return proxy;
         }
 
+        public void Relay(Stream relaySendingStream, Stream relayReceivingStream, object relayTarget = null)
```

Comment:
Please add detailed xml doc comments to this method, including a remarks section that explains the scenarios that it is meant to work with.